Sources/TuistGenerator/Generator/SchemeDescriptorsGenerator.swift

```diff
@@ -586,6 +586,11 @@ final class SchemeDescriptorsGenerator: SchemeDescriptorsGenerating {
             enableGPUFrameCaptureMode = .autoEnabled
         }
 
+        let disableGPUValidationMode = scheme.runAction?.metalOptions?.apiValidation == false
```

Comment:
```suggestion
        let disableGPUValidationMode = scheme.runAction?.metalOptions?.apiValidation ?? true
```

This should be equal now, no? If not, then let's revert the below as well, I think it would be good to have these written the same way

Reply:
gotcha. Let's just keep things as they are 🙂